src/triggers/commentDocument.ts

```diff
@@ -63,12 +64,24 @@ interface CommentsResponse {
 const getCommentList = () => async (z: ZObject, bundle: Bundle) => {
   const cursor = bundle.meta.page ? await z.cursor.get() : undefined;
 
-  const variables = {
+  const variables = omitBy({
     creatorId: bundle.inputData.creator_id,
     projectId: bundle.inputData.project_id,
     documentId: bundle.inputData.document_id,
     after: cursor,
-  };
+  }, v => v === undefined);
+
+  const filters = [];
+  if ("creatorId" in variables) {
+    filters.push(`{ user: { id: { eq: $creatorId } } }`);
+  }
+  if ("projectId" in variables) {
+    filters.push(`{ documentContent: { project: { id: { eq: $projectId }} } }`);
+    filters.push(`{ documentContent: { document: { project: { id: { eq: $projectId }}}}}`);
+  }
```

Comment:
@suevalov you added this filter, but this will never be true as content comment belongs to, can either be linked to a Project or a Document and not both. Or am I missing something?

Reply:
On separate note, I think it's unfortunate that we exposed document content in the API as it's an internal implementation detail, would be better if these would just be documents. Not easy for externals to grasp this stuff